samples/wifi/monitor/README.rst

```diff
@@ -126,6 +126,61 @@ To enable this feature, use the configuration overlay files :file:`overlay-net-c
 When the offline net capture feature is enabled, incoming IEEE 802.11 packets are routed to the offline storage over the net capture tunnel.
 These packets can then be analyzed using Wireshark.
 
+Wireshark decode as IEEE 802.11
+===============================
+
+The packets from the device are sent to the host over the net capture tunnel using IP in IP tunneling.
+The nRF70 Series device sends the IEEE 802.11 packets prepended with a custom heametadata) over the net capture tunnel to the host.
+To analyze the packets in Wireshark, the payload of the UDP packets must be dissected as IEEE 802.11 packets.
+
+This support is only available in Wireshark 4.3 (under development).
+A custom build of Wireshark from the latest sources is required.
+Once the custom build is installed, complete the following steps to dissect the payload of the UDP packets as IEEE 802.11 packets:
+
+1. Ensure Wireshark is compiled with Lua support.
+#. Open Wireshark and go to **Analyze** > **Decode As** > **+**, then select **UDP**.
```

Comment:
These are buttons in Wireshark? If so, please use: 

:guilabel:`button name`

@richabp please confirm first.

Reply:
@krish2718 to confirm as I am not familiar with Wireshark UI.

Reply:
It seems that these are Menu and Menu items, if so, then use, :guilabel:`menu name`

Reply:
Yes, these are the menu navigations.